examples/create-next-app/tsconfig.json

```diff
@@ -22,5 +22,5 @@
     }
   },
   "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules", "open-next.config.ts"]
```

Comment:
is this intentional? why? 😕

Reply:
(the same question applies to all the other apps too)

Reply:
Looks like you figured out: `import type { OpenNextConfig } from "@opennextjs/aws/types/open-next";` does not compile because `@opennextjs/aws` is not installed.

Hopefully that will not be needed soon when we share code with aws.

Reply:
ahhhh ok I see 😕